src/test/ui/imports/local-modularized-tricky-fail-1.stderr

```diff
@@ -5,10 +5,11 @@ LL | exported!();
    | ^^^^^^^^ ambiguous name
    |
    = note: ambiguous because of a conflict between a name from a glob import and a macro-expanded name in the same module during import or macro resolution
-note: `exported` could refer to the macro defined here
-  --> $DIR/local-modularized-tricky-fail-1.rs:5:5
+note: `exported` could refer to the macro imported here
```

Comment:
Likewise, "imported" is quite unfortunate.

Reply:
https://github.com/rust-lang/rust/pull/103760#discussion_r1009690345